docs/README.md

```diff
@@ -3,99 +3,100 @@
-  * [Framework Types](#framework-types)
-  * [Framework integration tests](#framework-integration-tests)
-  * [Providers](#providers)
-    + [framework-provider-aws-\*](#framework-provider-aws-)
-    + [framework-provider-local-\*](#framework-provider-local-)
-    + [framework-provider-kubernetes-\*](#framework-provider-kubernetes-)
-    + [framework-provider-azure-\*](#framework-provider-azure-)
-  * [Configuration and environments](#configuration-and-environments)
-    + [Booster configuration](#booster-configuration)
-    + [Environments](#environments)
-  * [Extending Booster with Rockets!](#extending-booster-with-rockets)
-    + [Naming recommendations](#naming-recommendations)
-    + [Infrastructure extensions](#infrastructure-extensions)
-    + [Runtime extensions (Not yet implemented)](#runtime-extensions-not-yet-implemented)
-    + [Deploy and init hooks (Not yet implemented)](#deploy-and-init-hooks-not-yet-implemented)
+  - [Contributing](#contributing)
+  - [Framework Core](#framework-core)
+  - [Framework Types](#framework-types)
+  - [Framework integration tests](#framework-integration-tests)
+  - [Providers](#providers)
+    - [framework-provider-aws-\*](#framework-provider-aws-)
+    - [framework-provider-local-\*](#framework-provider-local-)
+    - [framework-provider-kubernetes-\*](#framework-provider-kubernetes-)
+    - [framework-provider-azure-\*](#framework-provider-azure-)
+  - [Configuration and environments](#configuration-and-environments)
+    - [Booster configuration](#booster-configuration)
+    - [Environments](#environments)
+  - [Extending Booster with Rockets!](#extending-booster-with-rockets)
+    - [Naming recommendations](#naming-recommendations)
+    - [Infrastructure extensions](#infrastructure-extensions)
+    - [Runtime extensions (Not yet implemented)](#runtime-extensions-not-yet-implemented)
+    - [Deploy and init hooks (Not yet implemented)](#deploy-and-init-hooks-not-yet-implemented)
 - [Debugging and testing Booster applications](#debugging-and-testing-booster-applications)
-  * [Running Booster applications locally](#running-booster-applications-locally)
-    + [Local development prerequisites](#local-development-prerequisites)
-    + [Starting your application](#starting-your-application)
-    + [Performing Auth requests](#performing-auth-requests)
-    + [Performing GraphQL requests](#performing-graphql-requests)
-    + [Inspect Database information](#inspect-database-information)
-  * [Booster examples](#booster-examples)
+  - [Running Booster applications locally](#running-booster-applications-locally)
+    - [Local development prerequisites](#local-development-prerequisites)
+    - [Starting your application](#starting-your-application)
+    - [Performing Auth requests](#performing-auth-requests)
+    - [Performing GraphQL requests](#performing-graphql-requests)
+    - [Inspect Database information](#inspect-database-information)
+  - [Booster examples](#booster-examples)
```

Comment:
It's weird that you changed the format in the whole TOC. Are you using the linter config in `.markdownlintrc`? If you are, there might be some extra options we should add to the config to make sure we all use the same.

Reply:
I haven't change anything but if I'm the only one experiencing this, I'll love to compare our rules to find what I'm missing 😅